Taskfile.yml

```diff
@@ -0,0 +1,193 @@
+version: 3
+
+includes:
+  build: ./Taskfile_{{OS}}.yml
+
+vars:
+  BENCHTIME: 5s
+  FUZZTIME: 15s
+  FUZZCORPUS: ../fuzz-corpus
+
+tasks:
+  gen-version:
+    cmds:
+      - go generate -x ./internal/util/version
+
+  install-tools:
+    dir: tools
+    cmds:
+      - go mod tidy
+      - go mod verify
+      - go generate -x
+
+  init:
+    desc: "Install development tools"
+    deps: [gen-version, build:rm-bin]
+    cmds:
+      - go mod tidy
+      - task: install-tools
+
+  env-setup:
+    deps:
+      - gen-version
+    cmds:
+      - go run ./cmd/envtool/main.go
+
+  env-up-detach:
+    cmds:
+      - docker-compose up --always-recreate-deps --force-recreate --remove-orphans --renew-anon-volumes --detach
+
+  env-up:
+    desc: "Start development environment"
+    deps:
+      - env-up-detach
+      - env-setup
+    cmds:
+      - docker-compose logs --follow
+
+  env-pull:
+    cmds:
+      - docker-compose pull --include-deps --quiet
+
+  env-down:
+    desc: "Stop development environment"
+    cmds:
+      - docker-compose down --remove-orphans --volumes
+
+  gen:
+    desc: "Generate code"
+    cmds:
+    - ./bin/gofumpt
+    - go generate -x ./...
+    - task: fmt
+
+  fmt:
+    desc: "Format code"
+    cmds:
+#      - ./bin/gofumpt{{.EXT}}
+      - ./bin/gofumpt{{.EXT}} -w .
+
+  test:
+    cmds:
+      - go test -race -shuffle=on -coverprofile=cover.txt -coverpkg=./... ./...
+      - go test -race -shuffle=on -bench=. -benchtime=1x ./...
+
+
+  bench-short:
+    desc: "Benchmark for about 20 seconds (with default BENCHTIME)"
+    cmds:
+      - go test -list='Benchmark.*' ./...
+#      - {{.RM}} new.txt
+      # Running four functions for $(BENCHTIME) each..."
```

Comment:
That comment was a hack to echo a message with BENCHTIME substituted. I think we can remove it or print a message by some other mean.

Reply:
I've added echo to show BENCHTIME